torch/fx/experimental/proxy_tensor.py

```diff
@@ -520,6 +521,21 @@ def can_handle_tensor(x):
     else:
         constant = None
 
+    from .symbolic_shapes import compute_unbacked_bindings
+    # Can't use detect_fake_mode here,
+    #
+    # python test/distributed/_tensor/test_dtensor_compile.py -k
+    # test_tp_compile_fullgraph_is_seq_parallel_False
+    #
+    # will fail.  Very strange, it probably isn't right for them to be using
+    # two fake modes there...
+    fake_mode = torch._C._get_dispatch_mode(
+        torch._C._TorchDispatchModeKey.FAKE
+    )
+    if fake_mode and fake_mode.shape_env:
+        if symbol_to_path := compute_unbacked_bindings(fake_mode.shape_env, out):
+            proxy_out.node.meta["unbacked_bindings"] = symbol_to_path
```

Comment:
Maybe it would be nice if `set_example_value` was some shared piece of infra so we didn't need to duplicate metadata setting. This is probably way too much of a rabbithole though.

Reply:
Can't even use set_example_value here, proxy_tensor meta goes into val lol.